spec/addressable/uri_spec.rb

```diff
@@ -3511,22 +3553,6 @@ def to_s
         "http://user:pass@example.com/path/to/resource?query=x#fragment")
   end
 
-  it "should have the correct scheme after assignment" do
```

Comment:
Doh..it looks like i dropped more of this test than I should have. I'll pull it back and just drop the specific `bogus!` test.